src/storage/heap_file.c

```diff
@@ -11588,8 +11596,9 @@ heap_attrinfo_transform_to_disk_internal (THREAD_ENTRY * thread_p, HEAP_CACHE_AT
 
       if (ptr_varvals + mvcc_wasted_space >= buf->endptr)
 	{
-	  new_recdes->length = -expected_size;	/* set to negative */
-	  return S_DOESNT_FIT;
+	  // is it possible?
```

Comment:
I think we should add 
assert(false);
here  and bellow on ER_TF_BUFFER_OVERFLOW
We should try to fix unmatched get_packed_size vs pack

Reply:
Tests show that estimated size is imprecise and we'd probably have many crashes. We may consider this task for later.